src/wasm.rs

```diff
@@ -176,16 +176,16 @@ fn handle_module(
                 return Err(pretty_verifier_error(&context.func, fisa.isa, None, errors));
             }
         } else {
-            let (code_size, rodata_size) = context
+            let offsets = context
```

Comment:
nit: maybe rename `offsets` to `code_info`?

Reply:
Done.